Dockerfiles/ci/README.md

```diff
@@ -0,0 +1,37 @@
+# CVP upstream image for testing in github actions
+CVP upstream image operator bundle validate container
+
+# Usage
+
+## Building image on local machine 
+
+```
+git clone https://github.com/redhat-operator-ecosystem/operator-test-playbooks.git
+cd operator-test-playbooks
+
+### podman build -t <image_name>:<tagname> -f <filepath> --build-arg OPERATOR_SDK_VERSION=<operator_sdk_version>
+### example:
+podman build -t upstream_ci_image:latest -f Dockerfiles/ci/Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0
+```
+
+## Running tests locally with built image
+
+```
+// create empty directories for testing
+mkdir test_operator_work_dir output_logs
+podman run -it -v $PWD:/project/operator-test-playbooks -v ./Dockerfiles/ci/example-metadata-without-alm-annotations:/project/operator_dir -v ./output_logs:/project/output -v ./test_operator_work_dir:/project/test_operator_work_dir -e TEST_NAME=test_for_report_failed_empty_alm_examples -e OPERATOR_DIR=/project/operator_dir -e WORK_DIR=/project/output -e OPERATOR_WORK_DIR=/project/test_operator_work_dir/ upstream_ci_image:latest
```

Comment:
Suggest adding a comment here for consistency, like `// Run the test container. Artifacts will be saved in the mounted directories.`

Also, unless this is covered someplace else, it would be good to describe what each of the environment variables are for. Like, maybe explain the difference between OPERATOR_DIR and OPERATOR_WORK_DIR.